lib/events/athena/athena.go

```diff
@@ -37,6 +37,8 @@ import (
 	"github.com/gravitational/teleport/lib/backend"
 	"github.com/gravitational/teleport/lib/events"
 	"github.com/gravitational/teleport/lib/integrations/externalauditstorage"
+
+	"github.com/gravitational/teleport/lib/internal/context121"
```

Comment:
I don't see the corresponding files for the lib/internal/context121 package.

Reply:
That explains the CI failing...